pkg/kubectl/cmd/config/flags.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"github.com/spf13/cobra"
+
+	"k8s.io/kubernetes/pkg/kubectl/genericclioptions"
+	"k8s.io/kubernetes/pkg/printers"
```

Comment:
this dep won't do

Reply:
I was depending on it since `ToPrinter` returns a `printers.ResourcePrinter`. Do we want a copy of this interface ins genericclioptions as well?

Reply:
Additionally, PrintFlags#ToPrinter methods instantiate their respective printer before returning it, also resulting in a dependency on the pkg/printers package